pre_commit/languages/docker.py

```diff
@@ -84,16 +85,19 @@ def get_docker_user() -> str:  # pragma: win32 no cover
 
 
 def docker_cmd() -> Tuple[str, ...]:  # pragma: win32 no cover
-    return (
+    cmds = [
```

Comment:
this would be `args` or `cmd` not `cmds` -- but don't worry about this, do the other suggestion instead